### ISSUE TITLE
Freeze Range in SelectionRangeSlider

### ISSUE THREAD

**Issue by agregory96** [jupyter-widgets/ipywidgets#2832]
When using the SelectionRangeSlider, it would be nice to have a capability similar to Plotly's where you can click between the two current points set on the range slider and drag the bar to scroll over a that range. 

An example would be if the SelectionRangeSlider is over datetimes with a tick for each hour over multiple days. If you set the first point at 3pm on the first day and the second point at 3am on the second day, it would be nice to be able to click the slider between those two points and drag the selected range across the slider to see updates in 12-hour chunks (4pm-4am, 5pm -5am, etc.) as opposed to having to manually re-set the start and end points.

I realize this is really only applicable to functions sliding over time, but would still like to suggest it as a functionality. Alternatively, if anyone knows a workaround using click events or something similar, please let me know!

**Comment by jasongrout** [comment]
We don't have this turned on, but it is supported by the underlying slider: https://refreshless.com/nouislider/behaviour-option/

Perhaps we should either expose this property in the slider widgets so the user can select whether they want this behavior, or perhaps we should just turn on "drag-tap" behavior by default? @ibdafna, what do you think?

**Comment by jasongrout** [comment]
To make this the default behavior, we would just need to add a `behavior` option when we create the sliders at https://github.com/jupyter-widgets/ipywidgets/blob/b91a6d9844d1e4e2571f1e7d2fa5416abf468106/packages/controls/src/widget_int.ts#L206-L221 and https://github.com/jupyter-widgets/ipywidgets/blob/b91a6d9844d1e4e2571f1e7d2fa5416abf468106/packages/controls/src/widget_float.ts#L150-L164

**Comment by ibdafna** [comment]
> We don't have this turned on, but it is supported by the underlying slider: https://refreshless.com/nouislider/behaviour-option/
> 
> Perhaps we should either expose this property in the slider widgets so the user can select whether they want this behavior, or perhaps we should just turn on "drag-tap" behavior by default? @ibdafna, what do you think?

@jasongrout Indeed, that functionality is available, but not exposed. I am happy to have "drag-tap" available by default if you think it's reasonable. Will add a PR shortly.

**Comment by jasongrout** [comment]
> I am happy to have "drag-tap" available by default if you think it's reasonable. Will add a PR shortly.

I think that would be great! Like the original poster, I too have missed having this feature. Thanks for a PR!
